ros_ws/src/crazyswarm/src/crazyswarm_server.cpp

```diff
@@ -836,7 +784,7 @@ class CrazyflieGroup
     const std::vector<libobjecttracker::DynamicsConfiguration>& dynamicsConfigurations,
     const std::vector<libobjecttracker::MarkerConfiguration>& markerConfigurations,
     pcl::PointCloud<pcl::PointXYZ>::Ptr pMarkers,
-    std::vector<libmotioncapture::Object>* pMocapObjects,
+    std::map<std::string, libmotioncapture::RigidBody>* pMocapRigidBodies,
```

Comment:
if I understand correctly, `m_pMocapRigidBodies` will be dereferenced unconditionally if `m_useMotionCaptureObjectTracking` is true or `m_interactiveObject` is not an empty string. Might be good to check for non-null in the constructor (in a future commit, since this is a new feature not related to switching libraries).